api/tests/acceptance/application/password-controller_test.js

```diff
@@ -228,11 +234,84 @@ describe('Acceptance | Controller | password-controller', () => {
         });
       });
     });
+  });
+
+  describe('POST /api/expired-password-updates', () => {
+
+    const username = 'firstName.lastName0511';
+    const expiredPassword = 'Password01';
+    const newPassword = 'Password02';
+
+    const options = {
+      method: 'POST',
+      url: '/api/expired-password-updates',
+      payload: {
+        data: {
+          attributes: { username, expiredPassword, newPassword }
+        }
+      }
+    };
+
+    beforeEach(async () => {
+      databaseBuilder.factory.buildUser.withUnencryptedPassword({
+        username, rawPassword: expiredPassword, shouldChangePassword: true
+      });
+      await databaseBuilder.commit();
+    });
+
+    context('Success cases', () => {
+
+      it('should return 201 HTTP status code', async () => {
+        // when
+        const response = await server.inject(options);
+
+        // then
+        expect(response.statusCode).to.equal(201);
+      });
+    });
+
+    context('Error cases', () => {
+
+      it('should respond 404 HTTP status code when username does not exist', async () => {
+        // given
+        options.payload.data.attributes = { username: 'unknow', expiredPassword, newPassword };
+
+        // when
+        const response = await server.inject(options);
+
+        // then
+        expect(response.statusCode).to.equal(404);
+      });
```

Comment:
Pas très fan des contexts "Errors cases" avec plein de `it` dans lesquels les contexts seront en réalité différents. 
Je propose plutôt un truc du genre pour mieux mettre en relief l'intérêt des contexts

 ```suggestion
    context('when username does not exist', () => {

      // given
      options.payload.data.attributes = { username: 'unknow', expiredPassword, newPassword };

      it('should respond 404 HTTP status code', async () => {

        // when
        const response = await server.inject(options);

        // then
        expect(response.statusCode).to.equal(404);
      });
    });
```

Reply:
Potentiellement à appliquer aux tests qui suivent

Reply:
Done